### PR TITLE
Deprecating secure token functionality on May 18th.

### PR DESCRIPTION
**Pull request by astuyve**
https://github.com/ambethia/recaptcha/issues/179

R.I.P secure token.

Could possible put some warnings in if someone passes `secure_token`.

### CODE REVIEW

File: lib/recaptcha/client_helper.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@ def recaptcha_tags(options = {})
         a[k] = v if data_attributes.include?(k)
       end
       data_attributes[:sitekey] = public_key
-      data_attributes[:stoken] = Recaptcha::Token.secure_token if options[:stoken] != false
```
Comment (grosser) [inline lib/recaptcha/client_helper.rb]: blow up if `:stoken` is set to true ?

**Comment by grosser** [comment]
nice, also remove from readme ?

**Comment by astuyve** [comment]
Good points. I'll remove it from the readme and also raise an error if it's set.

**Comment by ur5us** [comment]
👍 Will that be merged and new version released any time soon? Currently adding `stoken: false` which will be obsolete once that's merged & released.

**Comment by astuyve** [comment]
I think the plan is to merge and release it tomorrow or on the 19th as per: https://github.com/ambethia/recaptcha/issues/179

**Comment by grosser** [comment]
was just waiting for the update ... did not check back / did not see the new commit ... so here goes nothing ...

**Comment by grosser** [comment]
2.0 🎉

**Pull request merged**